network/handlers/drain_test.go

```diff
@@ -193,7 +202,7 @@ func TestDrainMechanics(t *testing.T) {
 		// Expected.
 	}
 
-	mt.advance(61 * time.Millisecond)
+	mt.advance(60 * time.Millisecond)
```

Comment:
60 seems arbitrary now.  Perhaps use `timeout` and add a comment noting the significance of this advance?  e.g. moving past our trigger point?

Reply:
just used `epsilon`.